library/common-library/templates/_agent-config.tpl

```diff
@@ -0,0 +1,17 @@
+{{/*
+This helper should return the defaults that all agents should have
+*/}}
+{{- define "common.agentConfig.defaults" -}}
+{{- if include "common.verboseLog" . }}
+verbose: 1
+{{- end }}
+{{- if (include "common.nrStaging" . ) }}
+staging: true
+{{- end }}
```

Comment:
could we add as well NRIA_CUSTOM_ATTRIBUTES?

Reply:
I am in if we want custom attributes as a `global` to be honest...

Reply:
I think we will have the fattest global I've ever seen in a helm chart 😅 
But let's go for it, at this point we're already in.

Reply:
This conversation will be solved with #767